lib/src/checkers/minimum_tap_area_checker.dart

```diff
@@ -99,4 +97,19 @@ should be at least ${format(minTapArea)}x${format(minTapArea)}''',
         ? rounded.toStringAsFixed(0)
         : rounded.toStringAsFixed(2);
   }
+
+  /// Returns true if rectange of a node is (partially) off screen
+  bool _isNodeOffScreen(
+    Rect paintBounds,
+    FlutterView window, {
+    double delta = 10.0,
```

Comment:
`delta` param doesn't seem to be used, can be removed?

Reply:
It's used inside `_isNodeOffScreen()` but its value is never changed from outside so I made it a `const` inside this method's body